### PR TITLE
Issue/#12

### LINKED ISSUE #12: most of the page is still in english, translate everything!!!
english
image
i mean everywhere not just this page

### PR DESCRIPTION
**Pull request by ninadarbaidze**
closes #12

### CODE REVIEW

File: public/locales/en/movies.json [hunk 1]
```diff
@@ -1,21 +1 @@
...
-  "no": "No",
-  "required": "* Please fill all fields",
-  "sure": "Are you sure?",
-  "total": "Total",
-  "updateMovie": "Update movie",
-  "year": "Release year",
-  "yes": "Yes",
-  "zeroMovie": "There are no movies yet, but you can add some by clicking \"Add Movie\" button"
-}
+{"addbtn":"Add movie","addMovie":"Add movie","budget":"Budget","changeImg":"Change photo","chooseFile":"Choose file","descriptionH1":"Movie Description","director":"Director","drag":"Drag & drop your image here or","editMovie":"Edit Movie","genre":"Genre","heading":"My list of movies","no":"No","required":"* Please fill all fields","sure":"Are you sure?","total":"Total","updateMovie":"Update movie","year":"Release year","yes":"Yes","zeroMovie":"There are no movies yet, but you can add some by clicking \"Add Movie\" button"}
```
Comment (GigaGiorgadze) [inline public/locales/en/movies.json:1]: format.... every file

File: schema/quoteSchema.ts [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
-});
-
-export default quoteSchema;
+const useQuoteSchema = () => {
+  const { t } = useTranslation();
+
+  const quoteSchema = Yup.object({
+    quoteGE: Yup.string()
+      .required(`${t('home:commonRequired')}`)
+      .matches(/[\u10A0-\u10FF]/, 'წერე ქართულად'),
```
Comment (GigaGiorgadze) [inline schema/quoteSchema.ts]: why is this a hook? just pass home:commondRequired as ke and then use it in file, just overcomplicating

**Review by GigaGiorgadze — CHANGES_REQUESTED** [review]

File: schema/quoteSchema.ts (hunk 2 above)
Comment (ninadarbaidze) [reply schema/quoteSchema.ts]: I can't use t without react component, or you're saying something else?

Comment (GigaGiorgadze) [reply schema/quoteSchema.ts]: ik that what im saying just type required(home:commandRequired) and then translate it in component when error happens not here

Comment (ninadarbaidze) [reply schema/quoteSchema.ts]: oh I get it now

**Review by GigaGiorgadze — APPROVED** [review]

**Pull request merged**